Classes/Service/AfxService.php

```diff
@@ -309,10 +309,12 @@ protected static function astNodeListToFusion($payload, $indentation = '')
             return $astNode;
         }, $payload);
 
-        // filter empty text nodes
+        // filter empty text nodes and comments
         $payload = array_filter($payload, function ($astNode) {
             if ($astNode['type'] === 'text' && $astNode['payload'] == '') {
                 return false;
+            } elseif ($astNode['type'] === 'comment') {
```

Comment:
can be simplified to:
```
if (($astNode['type'] === 'text' && $astNode['payload'] == '') || $astNode['type'] === 'comment') {
    return false;
}

return true;
```

Reply:
Thank you for the feedback.

That is true and it shows you really looked over the code. However I considered the way this is written slightly better readable. 

As this code is run only once each deployment before the ast is changed this is also no performance issue.

If you tested this successfully a comment would be great. Would really like to have some feedback before merging

Reply:
ok then write it at least like this:

```
if (($astNode['type'] === 'text' && $astNode['payload'] == '')) {
    return false;
}

if ($astNode['type'] === 'comment') {
    return false;
}

return true;
```

Reply:
Adjusted reads much nicer as you suggested, thanks